src/components/FilterBox.svelte

```diff
@@ -0,0 +1,20 @@
+<script>
+  import throttle from "just-throttle";
+
+  export let onChangeText = null;
```

Comment:
Pretty sure this doesn't need to be set to null.

```suggestion
  export let onChangeText;
```